### ISSUE TITLE
It should be possible to select and copy text in multiple messages at once

### ISSUE THREAD

**Issue by special** [ricochet-im/ricochet#401]
@sugoiuguu mentions in #398:
> You're [not] able to select multiple lines of text at the same time, most likely because different text boxes are used for each message

**Comment by dsernst** [comment]
Yeah, this is a bit of a frustrating limitation. I'll explain my use-case:

I've having a great conversation with a friend... I want to save a bunch of our conversation to turn into a blog post. 

(He's totally 👍 with this, happy to get his permission first if there was a concern about privacy.)

Right now I have to manually copy every individual message (there's 30 of them), or maybe take screenshots and try to OCR into editable text later?
